python/doc/examples/meta_modeling/polynomial_chaos_metamodel/plot_chaos_cleaning_strategy.py

```diff
@@ -168,7 +221,9 @@
 
 # %%
 #
-# The following function takes a polynomial chaos result as input and prints a given maximum number of coefficients of this polynomial. It can take into account a threshold, so that we can avoid to print coefficients which are very close to zero.
+# The following function takes a polynomial chaos result as input and prints a 
+# given maximum number of coefficients of this polynomial. It can take into account 
+# a threshold, so that we can avoid to print coefficients which are very close to zero.
 
 # %%
 def printCoefficientsTable(
```

Comment:
In the docstring of `printCoefficientsTable`, there is a type: "stricly" instead of "strictly", but I cannot reach it with a GitHib suggestion.